### ISSUE TITLE
Bug: lexical eslint plugin broke in 0.16.0

### ISSUE THREAD

**Issue by AlessioGr** [facebook/lexical#6251]
`@lexical/eslint-plugin` version 0.15.0 works as expected (even though all other lexical packages are on 0.16.0).

The moment I upgrade it to 0.16.0, I'm getting this error when eslint runs:

```
alessio@Alessios-MacBook-Pro richtext-lexical % eslint .

Oops! Something went wrong! :(

ESLint: 8.57.0

ESLint couldn't find the config "plugin:@lexical/recommended" to extend from. Please check that the name of the config is correct.

The config "plugin:@lexical/recommended" was referenced from the config file in "/Users/alessio/Documents/GitHub/payload/packages/richtext-lexical/.eslintrc.cjs".

If you still have problems, please stop by https://eslint.org/chat/help to chat with the team.

alessio@Alessios-MacBook-Pro richtext-lexical % 
```

My .eslintrc:

```ts
/** @type {import('eslint').Linter.Config} */
module.exports = {
  parserOptions: {
    project: ['./tsconfig.json'],
    tsconfigRootDir: __dirname,
  },
  extends: [
    "plugin:@lexical/recommended"
  ],
}

```

**Comment by ivailop7** [comment]
@etrepum might have more context

**Comment by etrepum** [comment]
Fix for this is in #6252 - the change from auto to named default exports broke this (#6088). We needed to do that when deprecating default exports in @lexical/react because mixed exports are not allowed.

**Comment by etrepum** [comment]
You can pin @lexical/eslint-plugin at 0.15.0 for now, it does not have any direct dependencies on any other package
